lib/config/error-codes/index.js

```diff
@@ -89,13 +88,12 @@ function checkErrorCodes(errorCodes) {
   }
 }
 
-checkErrorCodes({ api, internal, external, protocols, plugins, entrypoints });
+checkErrorCodes({ internal, external, api, network, plugins });
 
 module.exports = {
-  api,
   internal,
-  entrypoints,
+  external,
+  api,
+  network,
   plugins,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)